### ISSUE TITLE
Events do not show on calendar

### ISSUE THREAD

**Issue by jware14** [mscrivo/ootd#2]
I downloaded OoTD and everything works fine, except my Office Outlook calendar events do not show on the desktop OoTD calendar.  Can anyone assist?  Jim

**Comment by mscrivo** [comment]
Are you using the default calendar in Outlook?  or did you create a custom one?  If you created a custom one, you'll have to select it, by right clicking on the tray icon and choosing "Select Folder"

**Comment by jware14** [comment]
Yes, I am using the default calendar in Outlook.  I have not created a custom calendar--just the default on loaded in Office 2013.

**Comment by mscrivo** [comment]
Would you be able to attach screenshots of both your calendar in Outlook
showing items (please block out any sensitive info) and OotD not showing
any items?

**Comment by jware14** [comment]
Attached care default Outlook calendar and desktop screenshots.  I suspect the problem is operator error, but darned if I can figure out what the error is.

 

Thanks

 

Jim Ware

 <mailto:Jware14@cox.net> Jware14@cox.net

Cell 405-315-4533

**Comment by mscrivo** [comment]
@jware14 you'll have to attach the screenshots directly to this issue in github, attaching them via email doesn't work.

**Comment by jware14** [comment]
Links below--hopefully they will work.  Thanks--Jim
![calendar on desktop](https://cloud.githubusercontent.com/assets/13141292/8584972/f32c3fb6-25a1-11e5-9ebe-f630e9c38dfb.PNG)
![default outlook calendar](https://cloud.githubusercontent.com/assets/13141292/8584978/fc911978-25a1-11e5-848f-d3f15b193781.PNG)

**Comment by mscrivo** [comment]
Thanks Jim, got the images and removed them from here to protect your email address.  Very strange, do you know if you account is a connected Exchange account?  or just regular POP3/IMAP email?

I'm really not sure what would cause this.

**Comment by jware14** [comment]
My email is through Cox and is a regular POP3/IMAP email account.  I am puzzled also, but would like to get this app working—looks very useful!  Jim

 

Jim Ware

 <mailto:Jware14@cox.net> Jware14@cox.net

Cell 405-315-4533

**Comment by mscrivo** [comment]
What happens if you right click on the tray icon for OotD and select another view like Inbox?  or better yet, try "Select Folder" and pick Calendar from there.  Any difference?

**Comment by jware14** [comment]
If I select another view like Inbox the messages show on the desktop and if Contacts are selected they also show.  Only the calendar events do not show.

 

Picking Calendar in the Select Folder menu does not make any difference.  I tried both calendars available (untitled and untitled-this computer only) and nothing changed.

 

Possibly a calendar events linkage error?

 

Jim

 

Jim Ware

 <mailto:Jware14@cox.net> Jware14@cox.net

Cell 405-315-4533

**Comment by mscrivo** [comment]
I'm really at a loss as to what to do with this one, I've not seen it before.  The only other thing I can think of is to try creating a new Calendar in Outlook and see if the items on that one show up in OotD.  If they do and you find it's worth the effort, move everything from your old calendar to the new one.

**Comment by jware14** [comment]
I tried a new calendar--no luck.  I also tried updating to the latest version and it bombed.  I uninstalled the software then reinstalled with the latest version and this time it still bombed.  Below is the error message I got when trying to complete the install.

![ootd error snip](https://cloud.githubusercontent.com/assets/13141292/8735686/56662c86-2bd8-11e5-8d68-ac50e3c9be67.PNG)

Does this shed any light on the problem?
